develop/EPM_Integration_Training/Web_Apps/EpmIntegrationTrainingDashboards.md

```diff
@@ -0,0 +1,11 @@
+---
+uid: EpmIntegrationTrainingDashboards
+---
+
+# EPM and Dashboards
+
+Dashboards created with the [Dashboards app](xref:newR_D) are often used in the EPM Solution.
+
+These frequently make use of EPM feeds. For more details on using EPM feeds, see [EPM feed](xref:DashboardCPEFeed).
+
+When you have deployed the *EPM Integration Training* package, the **Device Overview** dashboard will be available. This dashboard leverages an EPM feed and serves as an excellent reference model for designing dashboards across various EPM fields.
```

Comment:
What about the Device Grid Overview dashboard, which is mentioned as one of the components of the package?

Reply:
The Device Grid Overview dashboard is also included and installed and also leverages the EPM feed component.